src/GBA.c

```diff
@@ -529,7 +529,7 @@ static inline int dataTicksAccess16(u32 address) // DATA 8/16bits NON SEQ
     int waitState = value;
     if (!waitState)
       waitState = 1;
-    busPrefetchCount = ((++busPrefetchCount)<<waitState) - 1;
+    busPrefetchCount = ((busPrefetchCount + busPrefetchCount)<<waitState) - 1;
```

Comment:
Thanks for the comments, I am not really a practiced programmer so mistakes aren't unexpected...

So I applied these changes to a fresh copy of your repo.

With `busPrefetchCount = ((busPrefetchCount+1)<<waitState) - 1;`
014392564ae8244aad576b3f573ca25d  GBA.o

With `busPrefetchCount = ((busPrefetchCount + busPrefetchCount)<<waitState) - 1;`
858644e370ac25c2975c33ff0bfc757c  GBA.o

Original:
1110928ae2751e04dee890dd3c4ddea2  GBA.o

It seems either method of silencing the warning will change the md5sum of the object files.